python/cuml/tests/test_linear_model.py

```diff
@@ -972,3 +977,30 @@ def test_elasticnet_solvers_eq(datatype, alpha, l1_ratio, nrows, column_info):
     assert qn.score(X_test, cd_res) > 0.95
     # coefficients of the two models should be close
     assert np.corrcoef(cd.coef_, qn.coef_)[0, 1] > 0.98
+
+
+@given(
+    dataset=standard_regression_datasets(
+        n_features=st.integers(min_value=1, max_value=10),
+        dtypes=floating_dtypes(sizes=(32, 64)),
+    ),
+    algorithm=algorithms,
+    xp=st.sampled_from([np, cp]),
+    copy=st.booleans(),
+)
+@example(make_regression(n_features=1), "svd", cp, True)
+@example(make_regression(n_features=1), "svd", cp, False)
+@example(make_regression(n_features=1), "svd", np, False)
```

Comment:
why both `cp` and `np` examples? Shouldn't be needing both since the testing of formats is in the input functions testing.

Reply:
Yeah that's true after some careful consideration